src/index.js

```diff
@@ -427,7 +436,7 @@ function isArray (arr) {
 }
 
 
-function addHole(contour, idx) {
+function addHole(contour, idx) { // eslint-disable-line no-unused-vars
```

Comment:
There's not much utility to keep these un-used methods, there's a reason for this `eslint` warning/error.